src/components/card/CartCard.jsx

```diff
@@ -0,0 +1,76 @@
+import axios from 'axios';
+import React from 'react'
+import { useAthorizer } from '../../context/AuthorizerContext';
+
+function CartCard({product}) {
+  
+    const {_id , name , rating , orignalPrice , discount , imageUrl , qty} = product;
+    const { authState , authDispatch } = useAthorizer();
+
+    const productQtyHandler = async( productId , type ) => {
+        try {
+         const response = await axios.post(`/api/user/cart/${productId}`,{action:{type:`${type}`}},{headers:{authorization :authState.encodedToken }})   
+            authDispatch({ type:"QUANTITY" , payload : response.data.cart})
+        } catch (error) {
+            console.log(error);
+        }
+    }
+
+    const removeFromCart = async(productId) =>{
+        try {
+                const response = await axios.delete(`/api/user/cart/${productId}`,{headers:{authorization :authState.encodedToken }})
+                authDispatch({type:"REMOVE_FROM_CART" , payload:response.data.cart});
+        } catch (error) {
+            console.log(error);
```

Comment:
Suggestion: Can use `console.error(error)` for error

Reply:
yes ,i will change it